services/core-web/common/actionCreators/permitActionCreator.js

```diff
@@ -286,3 +286,24 @@ export const updatePermitCondition = (permitConditionGuid, permitAmdendmentGuid,
     .catch(() => dispatch(error(reducerTypes.UPDATE_PERMIT_CONDITION)))
     .finally(() => dispatch(hideLoading()));
 };
+
+export const patchPermit = (permitGuid, mineGuid, payload) => (dispatch) => {
+  dispatch(request(reducerTypes.PATCH_PERMIT));
+  dispatch(showLoading());
+  return CustomAxios()
```

Comment:
showLoading is not scoped to a modal, but hideLoading is. Update depending where you want the loading indicator to be, I would assume modal.

Reply:
chatted via video. resolved.